dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerRelationshipsExportControllerTest.java

```diff
@@ -474,21 +482,26 @@ void getRelationshipsByTrackedEntityWithEnrollments()
     }
 
     @Test
-    void getRelationshipsByTrackedEntityWithAttributes()
+    void getRelationshipsByTrackedEntityAndEnrollmentWithAttributes()
```

Comment:
is there a reason why you don't follow the convention should...When...?

Reply:
My reason is that all other tests are named this way. They also clearly describe what the test is about IMHO. To me should/when would just make the test name longer without making it clearer. I can rename all tests if you think we should 😄 

How would you name this one?